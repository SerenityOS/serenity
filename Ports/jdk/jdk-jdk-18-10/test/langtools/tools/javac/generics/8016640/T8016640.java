/*
 * @test /nodynamiccopyright/
 * @bug     8016640 8022508
 * @summary compiler hangs if the generics arity of a base class is wrong
 * @compile/fail/ref=T8016640.out -XDrawDiagnostics T8016640.java
 */
class T8016640 {
    static class Foo<X,Y> { }
    static class BadFoo<T> extends Foo<T> { }
    static class SubBadFoo<T> extends BadFoo<T> { }
}
