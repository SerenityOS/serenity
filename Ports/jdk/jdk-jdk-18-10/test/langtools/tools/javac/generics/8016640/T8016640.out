T8016640.java:9:39: compiler.err.wrong.number.type.args: 2
1 error
